src/openforms/formio/service.py

```diff
@@ -86,3 +88,8 @@ def update_default_file_types(component: Component, **kwargs) -> None:
     if component.get("type") == "file" and component.get("useConfigFiletypes"):
         config = GlobalConfiguration.get_solo()
         component["filePattern"] = ",".join(config.form_upload_default_file_types)
+
+
+def update_content_inline_csp(component: Component, request: Request) -> None:
+    if component.get("type") == "content":
+        component["html"] = post_process_html(component["html"], request)
```

Comment:
I'd probably extend this post processor to accept an allow-list of CSS attributes to extract - reason being is that content components are not purely "trusted" content from form-designers, but can contain malicious user input if the form designer users variables inside the HTML (and the form submission data is passed as template context to those HTML blobs).

This should all be escaped by default by Django, but it's possible form designers disable auto-escape.

Reply:
Cool, I added a PR with some tinycss2.